docs/cloud/managing-airbyte-cloud/manage-airbyte-cloud-notifications.md

```diff
@@ -2,26 +2,35 @@
 
 This page provides guidance on how to manage notifications for Airbyte Cloud, allowing you to stay up-to-date on the activities in your workspace. 
 
-## Set up Webhook notifications
+
+## Configure Notification Settings
 
 To set up Webhook notifications:
 
 1. On the [Airbyte Cloud](http://cloud.airbyte.com) dashboard, click **Settings**.
 
 2. Click **Notifications**.
 
-3. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
+3. Have a Slack Webhook ready if you plan to use Slack Webhook notification. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
```

Comment:
```suggestion
3. Have a Slack Webhook ready if you plan to use Slack Webhook notifications. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).
```

Reply:
A webhook outside of Slack is also acceptable. Could we make this more generic?

Could rephrase to: 3. Have a webhook URL ready if you plan to use webhook notifications. Using a Slack webook is recommended. [Create an Incoming Webhook for Slack](https://api.slack.com/messaging/webhooks).

Reply:
done!  one thing to note is the webhook notification we sent might not be consumed by other endpoints other than slack; slack requires a json format when sending webhook message, though it's not strange nor specific to slack;

Reply:
Ah, okay good to know!